_posts/memory-creation-overflow-bug.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Solidity Memory Array Creation Overflow Bug"
+date:   2020-04-02
+author: Solidity Team
+---
+
+# Solidity Memory Array Creation Overflow Bug Announcement
+
+A bug in the Solidity code generator was reported through the <a href="https://bounty.ethereum.org/" target="_blank">Ethereum
+Foundation Bounty program</a>, by [[[Insert Name]]]. The bug is fixed with version 0.6.5, released on [[[Insert Release Date]]].
+
+## Who should be concerned
+
+If you have deployed contracts which allocates a memory array of user-supplied length, but does not copy or iterate over it, this will lead to memory corruption. In particular, a sufficiently large length during the allocation of the memory array will result in subsequent memory allocations to overlap with the memory region of that memory array.
```

Comment:
```suggestion
If you have deployed contracts which allocates a memory array of user-supplied length, but does not copy or iterate over it, this can lead to memory corruption. In particular, a sufficiently large length during the allocation of the memory array will result in subsequent memory allocations to overlap with the memory region of that memory array.
```